client/Cargo.toml

```
@@ -8,4 +8,8 @@ path = "src/main.rs"
 [dependencies]
 clap = { version = "4.5.26", features = ["derive"] }
 common = { path = "../common" }
-tokio = { version = "1.43.0", features = ["full"] }
\ No newline at end of file
+confy = "0.6.1"
+serde = "1.0.217"
+serde_derive = "1.0.217"
+tokio = { version = "1.43.0", features = ["full"] }
+url = "2.5.4"
```

serde対応してるっぽい

https://docs.rs/url/latest/url/

```suggestion
url = { version = "2.5.4", features = ["serde"] }
```